### PR TITLE
Switch to PAT generated GH token

### PR DESCRIPTION
**Pull request by ev1yehor**
## What does this PR do?
Switch to PAT generated GH token

## Checklist

- [ ] I have added test packages to [`test/packages`](https://github.com/elastic/package-spec/tree/main/test/packages) that prove my change is effective.
- [ ] I have added an entry in [`spec/changelog.yml`](https://github.com/elastic/package-spec/blob/main/spec/changelog.yml).

## Related issues

- https://github.com/elastic/ingest-dev/issues/3495

### CODE REVIEW

File: .buildkite/hooks/pre-command [hunk 1]
```diff
@@ -4,19 +4,15 @@ source .buildkite/scripts/tooling.sh
...
-    GITHUB_USERNAME_SECRET=$(retry 5 vault kv get -field username ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_USERNAME_SECRET
-    GITHUB_EMAIL_SECRET=$(retry 5 vault kv get -field email ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_EMAIL_SECRET
-    GITHUB_TOKEN=$(retry 5 vault kv get -field token ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_TOKEN
+    GITHUB_USERNAME_SECRET="elasticmachine"
+    export GITHUB_USERNAME_SECRET=$GITHUB_USERNAME_SECRET
+    export GITHUB_EMAIL_SECRET="elasticmachine@elastic.co"
+    export GITHUB_TOKEN=$VAULT_GITHUB_TOKEN
```
Comment (mrodm) [inline .buildkite/hooks/pre-command:17]: If ephemeral tokens are used, maybe it's not needed those environment variables.

Are those ephemeral tokens already available for `git` commands ?

If they are, we could try to remove these functions (and respective calls) and test it:
https://github.com/elastic/package-spec/blob/7aa453907fb0406e32dca37e296e0a2199416487/.buildkite/scripts/test-with-integrations.sh#L56-L67


The same would happen for the elastic-package: https://github.com/elastic/elastic-package/pull/1942

**Comment by ev1yehor** [comment]
test integrations

**Comment by ev1yehor** [comment]
test integrations

File: .buildkite/hooks/pre-command (hunk 1 above)
Comment (ev1yehor) [reply .buildkite/hooks/pre-command:17]: I tried to remove this git config lines and test it, but it is not work and ask to set this parameters.
So I will leave it like it is for now.

**Comment by ev1yehor** [comment]
Changes was tested. https://buildkite.com/elastic/package-spec-test-with-integrations/builds/43#0190df57-e9b7-4bc6-ae17-8a7cc3aac072
Once checks will be green, can be reviewed.

**Review by jsoriano — APPROVED** [review]
Thanks @ev1yehor!

**Pull request merged**